src/dawn/SIR/CMakeLists.txt

```diff
@@ -12,25 +12,51 @@
 ##
 ##===------------------------------------------------------------------------------------------===##
 
+include(DawnProtobufGenerate)
+
+# Genreate C++ proto files
+dawn_protobuf_generate(
+  OUT_FILES sir_proto_cpp_files
+  OUT_INCLUDE_DIRS sir_proto_include_dirs
+  PROTOS ${CMAKE_CURRENT_SOURCE_DIR}/SIR.proto
+  LANGUAGE cpp
+)
+include_directories(SYSTEM ${sir_proto_include_dirs})
+
 dawn_add_library(
   NAME DawnSIR
-  SOURCES AST.h  
+  SOURCES AST.h
           AST.cpp
-          ASTExpr.cpp          
+          ASTExpr.cpp
           ASTExpr.h
           ASTFwd.h
-          ASTStmt.cpp          
+          ASTStmt.cpp
           ASTStmt.h
           ASTStringifier.cpp
           ASTStringifier.h
           ASTUtil.cpp
           ASTUtil.h
-          ASTVisitor.cpp  
+          ASTVisitor.cpp
           ASTVisitor.h
           SIR.cpp
           SIR.h
-          SIRSerializerJSON.cpp
-          SIRSerializerJSON.h          
+          SIR.proto
+          SIRSerializer.h
+          SIRSerializer.cpp
           Statement.h
+          ${sir_proto_cpp_files}
   OBJECT
 )
+
+# Genreate python proto files
+dawn_protobuf_generate(
+  OUT_FILES sir_proto_python_files
```

Comment:
Where do we get this from?

Reply:
This is generated by `protoc` compiler from `SIR.proto`